app/policies/linked_profile_policy.rb

```diff
@@ -0,0 +1,11 @@
+class LinkedProfilePolicy < ApplicationPolicy
+  def show?
+    record.user == user || user.has_role?(:admin)
```

Comment:
aye aye captain! 

(now a bit off-topic) why do you do this then?
```
class LibraryEntryPolicy < ApplicationPolicy
  def show?
    # Yes, if you own it or are admin
    return true if record.user == user || user.has_role?(:admin)
    # No, if it's private
    return false if record.private?
    # No, if it's nsfw and you're not a perv
    return false if user.sfw_filter? && record.media.nsfw?
    # Otherwise, yeah
    true
  end
```